packages/apps/desktop-viewer-test/src/frontend/extensions/IModelMergeStatusBarItem.tsx

```diff
@@ -97,22 +116,15 @@ const MergeStatusBarItem = () => {
     }
   }, [iModelConnection, isMounted]);
 
-  if (!accessToken && mergeStatus !== ModelStatus.SNAPSHOT) {
-    return (
-      <div title="Click to login and view the model's status">
-        <SvgUser
-          onClick={onLoginClick}
-          className="model-status actionable status-bar-status"
-        />
-      </div>
-    );
-  }
-  return (
-    <BriefcaseStatus
-      mergeStatus={mergeStatus}
-      onMergeClick={onMergeClick}
-      className={"status-bar-status"}
-    />
+  return mergeStatus === ModelStatus.SNAPSHOT ? null : (
+    <div className="status-bar-status">
+      <span className="status-label">Changes: </span>
```

Comment:
localize

Reply:
fixed

Reply:
could you add the localization fixes to the template as well?